lib/PRC/Secrets.pm

```diff
@@ -37,6 +37,15 @@ sub client_secret {
   return shift->_read('CLIENT_SECRET');
 }
 
+=head2 init_admin
```

Comment:
Let's not do the init_admin thing. I'd rather manually assign admin roles for now.